modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc

```diff
@@ -0,0 +1,372 @@
+// Module included in the following assemblies:
+//
+// * rosa_planning/rosa-sts-ocm-role.adoc
+// * rosa_architecture/rosa-sts-about-iam-resources.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-sts-aws-requirements-attaching-boundary-policy_{context}"]
+= Permission Boundaries for the Installer Role
+
+You can apply a policy as a _permissions boundary_ on a role.
+You can use an AWS-managed policy or a customer-managed policy to set the boundary for an IAM entity (user or role). The policy limits the maximum permissions for the user or role. ROSA includes a set of three prepared permission boundary policy files, with which you can restrict permissions for the installer role since changing the installer policy itself is not supported.
+
+
+The boundary policy files are as follows:
+
+* The _core_ boundary policy file contains the minimum permissions needed for ROSA (classic architecture) installer to install a cluster.
+If you need to install a cluster with the most restricted permissions, only use this permission boundary policy and there would be no VPC or PrivateLink (PL) permissions for the installer role.
```

Comment:
I don't think this suggestion reads quite right. The core policy should be used by itself if the customer is providing a VPC (BYO-VPC), and does not need PrivateLink. All cluster installs have a VPC, it will either need to be provided by the customer, OR created by the installer (which when using a permission boundary would require the VPC permission set + core permission set).

Reply:
what do you suggest @fahlmant please

Reply:
discussed in slack and came up with new content